devicetypes/smartthings/orvibo-gas-detector.src/Orvibo-Gas-detector.groovy

```diff
@@ -94,6 +94,7 @@ def ping() {
 }
 def configure() {
 	log.debug "configure"
-	//The gas heart beat is 30 mintues, so we set double time for one package loss.
-	sendEvent(name: "checkInterval", value: 30 * 2 * 60 + 2*60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	//The gas sensor has no battery , it can response the ping immediately
+	//so i set the checkTinterval to 5 minutes .
+	sendEvent(name: "checkInterval", value: 5*60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
Health Check is not meant to be used as a polling service. It's meant to verify that devices have not gone offline. Additionally, what user experience is improved by polling the device every five minutes? The device should be notifying us of its changes. If it's not, there should be configuration options to adjust the reporting interval.

Reply:
The tester said the 30 minutes is too long to be offline when the gas is remove.
The test case need the gas sensor to be offline in five minutes when it is removed from the hub.

Reply:
That requirement should be adjusted. Most of our sensors have a 32 minute health check and should be marked offline after 33 minutes of being unplugged or taken out of range.